### ISSUE TITLE
Fix inaccessible text colour combinations on Dark Mode

### ISSUE THREAD

**Issue by travis-jeans** [neocities/neocities#658]
The colours in the header, links, user's site name, and pagination in dark mode have below <3:1 contrast, when text should have [a minimum of 4.5:1 (WCAG) ](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) to be accessible.

The lightest colour of the gradient `#315f6b` and `#ff6b82` has a contrast ratio of 2.6:1. Please see the [WebAim colour contrast analyser to fix it](https://webaim.org/resources/contrastchecker/?fcolor=315F6B&bcolor=FF6B82).

<img width="243" height="45" alt="The Neocities mascot beside the Websites and Search button with the top-down dark green gradient background and pink text" src="https://github.com/user-attachments/assets/280b43b9-ebd3-4e9b-84a7-4ab50792774e" />

**Comment by kyledrake** [comment]
Should be fixed, thanks!

**Comment by travis-jeans** [comment]
Thank you for this, greatly appreciated.

Just wanted to let you know there's an area that was missed, the visited link style has below 3:1 contrast still. Visited links are using a colour with `#e93250`, a dark pink. Since the pink used for default links is around 6.3:1, it probably just doesn't need to be as dark a colour.

Example:
<img width="470" height="254" alt="Screenshot of Sites you Follow link with the visited link style class." src="https://github.com/user-attachments/assets/f32ec186-de57-4fcc-97ce-89afaf0dea9c" />
